### ISSUE TITLE
Lesson 8 - 14 (Quiz: Softmax) text issue

### ISSUE THREAD

**Issue by ibrasho** [udacity/sdc-issue-reports#217]
> In the one dimensional case, the array is just a single set of logits. **The one dimensional array is a single set of logits.**
> In the two dimensional case, each column in the array is a set of logits.
> **The one dimensional array is a single set of logits.The one dimensional array is a single set of logits.** The softmax(x) function should return a NumPy array of the same shape as x.

I'm assuming the bolded text is a duplicate, so it should be:
> In the one-dimensional case, the array is just a single set of logits. 
> In the two-dimensional case, each column in the array is a set of logits.
> The softmax(x) function should return a NumPy array of the same shape as x.

**Comment by ibrasho** [comment]
> Given a two-dimensional array where each column represents a set**The one dimensional array is a single set of logits.** of logits:

Should be:
> Given a two-dimensional array where each column represents a set of logits:

**Comment by Brok-Bucholtz** [comment]
@ibrasho 

Thanks for letting us know.  We removed the extra lines of "The one dimensional array is a single set of logits".  Let use know if you see anything else.
